src/lib/features/integration-events/integration-events.e2e.test.ts

```diff
@@ -5,14 +5,12 @@ import {
 } from '../../../test/e2e/helpers/test-helper';
 import getLogger from '../../../test/fixtures/no-logger';
 import { TEST_AUDIT_USER } from '../../types';
-import type {
-    IntegrationEventsStore,
-    IntegrationEventWriteModel,
-} from './integration-events-store';
+import type { IntegrationEventsService } from './integration-events-service';
```

Comment:
So this gets swapped with the service. Kinda suggests that the store is doing all the work here, is there a plan to have the service own a responsibility in a future PR?

Reply:
Yeah, the idea is that this e2e test grows along with the development of this feature. We only had the store before, so we used that. We now have the service, which should be responsible for the business logic of the feature, so I'm using the service instead. 

The service isn't really doing much. That may change in the future or it may not, but if it does we now have a place for it 🤷 . Most of our services are just calling store methods so this is at least consistent in that sense.

I guess one thing that it is doing for now is controlling the clean up behavior through the feature flag. By checking it in the `cleanUpEvents` method, we can always schedule the method but dynamically control whether we're actually cleaning up events (see: https://github.com/Unleash/unleash/pull/5363). I don't think there's an outer layer that is more suitable for this.